astro/src/content/articles/authentication/how-passwordless-works.mdx

```diff
@@ -0,0 +1,151 @@
+
+### How Does Passwordless Benefit UX?
+
+Passwordless provides significant user experience (UX) benefits by helping users sign in quickly with minimal effort. Instead of having to remember and type out complex passwords, users can simply click a link, scan their fingerprint, or acknowledge a prompt sent to one of their devices.
+
+Many people struggle to manage their passwords or are concerned that they may forget or misplace them. The widening understanding of password issues also means some users will have a peripheral awareness that passwords can be insecure, which could induce hesitance to sign up for new services that use them. Passwordless removes these doubts from the experience, helping users feel comfortable when interacting with your platform.
+
+Furthermore, passwordless factors are constantly accessible to the user, as opposed to passwords, which may need to be referenced from a password manager every time they're used. Constantly looking up passwords creates friction that can lead to session abandonment when the user tries to authenticate from a different device and can't access their saved passwords.
+
+### Passwordless UX Challenges
+
+Not that passwordless is a panacea for all authentication issues. Although it can remove login roadblocks, it also has the potential to *create* them or frustrate users who dislike particular methods.
+
+For example, biometric authentication might satisfy some people while deterring others who don't have a compatible device or don't want to use it. Similarly, users may find it distracting if they have to repeatedly switch to their email inbox to complete authentication attempts, so several different options should ideally be offered.
+
+Another pitfall is that poorly implemented passwordless is often slower than using passwords under ideal conditions. When a password is saved in a browser, users can normally authenticate themselves almost instantly. With passwordless, you can expect a small delay---whether while waiting for an email to arrive or using your device's hardware to complete a biometric scan.
+
+It's therefore important to consider ways of mitigating these problems, such as by clearly communicating how to complete the process and allowing a quick switch to a different authentication method if the first one gets stuck or times out.
+
+## Methods of Passwordless Authentication
+
+There's no single way to implement passwordless authentication. The only criterion is that no passwords are used. However, most services have gravitated towards offering one or more of the following methods:
+
+- **Biometric authentication:** a fingerprint or face scan
+
+- **Magic links:** a unique sign-in link sent to your email inbox
+
+- **Hardware tokens/keys:** using a dedicated security device
+
+Each option comes with benefits, drawbacks, and ideal use cases, so you should carefully consider the pros and cons of different methods before choosing which you'll support.
+
+### Biometric Authentication
```

Comment:
We're missing Passkeys all together in this article. We used to refer to biometric authentication as webauthn/passkeys because it uses the native device biometrics from hardware like phones/computers. But this goes really far down biometric angle, and doesn't at all mention or link to passkeys.